content/index.adoc

```diff
@@ -25,19 +25,30 @@ In alteration to the regular https://www.asam.net/license[license terms], ASAM a
 ifndef::use-antora-rules[]
 
 [preface]
-include::general_docs/foreword.adoc[leveloffset=+1]
+include::general-docs/foreword.adoc[leveloffset=+1]
 [preface]
-include::general_docs/introduction.adoc[leveloffset=+1]
+include::general-docs/introduction.adoc[leveloffset=+1]
 
 :sectnums:
-include::general_docs/scope.adoc[leveloffset=+1]
-include::general_docs/normative_references.adoc[leveloffset=+1]
-include::general_docs/abbreviations.adoc[leveloffset=+1]
-include::general_docs/backward_compatibility.adoc[leveloffset=+1]
-include::terms_and_definitions/terms_and_definitions.adoc[leveloffset=+1]
-include::use_cases/use_cases.adoc[leveloffset=+1]
-include::geometry/geometry-index.adoc[leveloffset=+1]
-include::material/general-structure-index.adoc[leveloffset=+1]
+include::general-docs/scope.adoc[leveloffset=+1]
+include::general-docs/normative-references.adoc[leveloffset=+1]
+include::general-docs/abbreviations.adoc[leveloffset=+1]
+include::general-docs/backward-compatibility.adoc[leveloffset=+1]
+include::terms-and-definitions/terms-and-definitions.adoc[leveloffset=+1]
+include::use-cases/use-cases.adoc[leveloffset=+1]
+Geometry
+include::geometry/introduction.adoc[leveloffset=+2]
```

Comment:
You changed the leveloffset here and are not using the index page anymore. Are you sure this works? Because locally (just in VS Code) this is not displayed correctly.

Also the "Geometry" one line above seems a bit out of place.

Reply:
The chapter structure in the index.adoc file has no influence on the antora build. It is there to create a local preview in VS Code for authors to check how there topic might look like in the html output. However, it will never resemble antora 100%, as it can only reference files that are stored in the repo (therefore, e.g. metadata.adoc cannot be shown.) Also, images will not be shown correctly as antora expects them in another folder than asciidoctor.
I improved the index.adoc file a little more so the preview looks prettier now.